src/graphql/candyMachine.ts

```diff
@@ -398,23 +606,41 @@ export const CandyMachineUploadMutation = mutationField('candyMachineUpload', {
       await fs.mkdir(CACHE_PATH);
     }
 
-    runUploadV2(logger, processId, args)
-      .catch((err) => {
-        logger.error('Aborting due to error');
-        logger.error(err);
-      })
-      .finally(async () => {
-        logger.info('Cleaning up');
-        await new Promise<void>((resolve, reject) => {
-          rimraf(`${storageDir}/${processId}`, (err) => {
-            if (err) {
-              reject(err);
-            }
-            resolve();
+    if (args.useHiddenSettings) {
+      runUploadV2UsingHiddenSettings(logger, processId, args)
+        .catch((err) => {
+          logger.error('Aborting due to error');
+          logger.error(err);
+        })
+        .finally(async () => {
+          logger.info('Cleaning up');
+          await new Promise<void>((resolve, reject) => {
+            rimraf(`${storageDir}/${processId}`, (err) => {
+              if (err) {
+                reject(err);
+              }
+              resolve();
```

Comment:
This repeated code is relatively complex. Good candidate for a simple function

Reply:
done